staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/validator.go

```diff
@@ -223,7 +224,7 @@ func (v *CELValidator) Validate(versionedAttr *generic.VersionedAttributes, vers
 			continue
 		}
 		t1 := time.Now()
-		evalResult, _, err := compilationResult.Program.Eval(va)
+		evalResult, _, err := compilationResult.Program.ContextEval(ctx, va)
```

Comment:
I think this might benefit from a simple unit test that the `ctx` passed into the top level controller interface is the same as/descendent of the one passed to the top-level controller/plugin's `Validate`. 

Just as a sanity check that they are in fact cancellable if the evaluator is cancellable.

Reply:
+1 to this testing approach

Reply:
Sorry for missing this comment earlier. I am exploring to see where the test should be added. Any suggestion?

Reply:
Added. Thanks